lib/tracking/utils/getCartItemTrackingData.js

```diff
@@ -1,39 +1,31 @@
-import routes from "routes";
-import { decodeOpaqueId } from "lib/utils/decoding";
-
 /**
  * Transform a CartItem object into a partial representation of the Segment product schema
  * @name getCartItemTrackingData
  * @param {Object} cartItem Object of the `CartItem` type
+ * @param {Object} locale String that allow us to construct the URL of the product
  * @returns {Object} Data suitable for tracking a `CartItem`
  */
-export default function getCartItemTrackingData(cartItem) {
+export default function getCartItemTrackingData(cartItem, locale) {
   const variant = { ...cartItem };
-  let url;
-
-  const { id } = decodeOpaqueId(variant._id);
-  const route = routes.findAndGetUrls("product", { slugOrId: variant.productSlug || id });
-
-  if (route && route.urls) {
-    url = route.urls.as;
-  }
-
   let cartId = {};
+
   if (variant.cart_id) {
     cartId = { cart_id: variant.cart_id }; // eslint-disable-line camelcase
   }
 
   return {
     ...cartId,
-    product_id: variant.productConfiguration.productVariantId, // eslint-disable-line camelcase
```

Comment:
Here we were using the ProductVariantId but I think the right value is productId